pkg/controller/discovery/web/plan.go

```diff
@@ -599,16 +599,72 @@ func (t *PlanTree) addMigrationPodsForCluster(cluster model.Cluster, migration *
 				continue
 			}
 		}
-		parent.Children = append(
-			parent.Children,
-			TreeNode{
+
+		node := TreeNode{
+			Kind:        migref.ToKind(m),
+			ObjectLink:  PodHandler{}.Link(&cluster, m),
+			Namespace:   m.Namespace,
+			Name:        m.Name,
+			ClusterType: t.clusterType(&cluster),
+		}
+
+		_, isStagePod := object.Labels[migapi.StagePodLabel]
+		if isStagePod {
+			t.addPVCsForPod(m, &node)
+		}
+
+		parent.Children = append(parent.Children, node)
+	}
+	return nil
+}
+
+//
+// Add PVCs related to a pod
+func (t *PlanTree) addPVCsForPod(pod *model.Pod, parent *TreeNode) error {
+	cluster := t.cluster.source
```

Comment:
Hmm, just realizing this is an odd assumption. We could also have stage pods on the target cluster. Will fix.

Reply:
@djwhatle Are you done changing this?

Reply:
Not yet, sorry I realized the problem right after I requested review. Will ping you again later.